packages/lib/src/vis/shared/VisGroup.tsx

```diff
@@ -0,0 +1,31 @@
+import type { ReactNode } from 'react';
+
+import { useAxisSystemContext } from './AxisSystemProvider';
+
+interface Props {
+  children: ReactNode;
+}
+
+function VisGroup(props: Props) {
```

Comment:
I've started using `interface Props {}` and then `props: PropsWithChildren<Props>` for cases like this.